### DISCUSSION TITLE
Proposal for a Specialized MySQL Docker Container for OpenMage

### DISCUSSION THREAD

**Discussion by samex** [OpenMage/magento-lts#3959] [Ideas]
Hello OpenMage Community,

I've been working with OpenMage LTS and recently discovered the PHP Docker container provided in the 'docker-php' repository. This container is a fantastic resource, offering an optimized environment tailored specifically for OpenMage.

Building on this, I propose the creation of a similar, specialized Docker container for MySQL, specifically optimized for use with OpenMage. This would aim to streamline setup, improve performance, and ensure greater compatibility and security for users deploying OpenMage in a Dockerized environment.

I believe this could include:

Pre-configured MySQL settings optimized for OpenMage.
Enhanced security configurations tailored for production deployments.
Possible integration features with the existing docker-php setup.
I'm very interested to hear your thoughts on this idea and would appreciate any feedback. Moreover, if anyone has been using a custom MySQL setup with OpenMage in Docker and would like to share insights or configurations, it could greatly inform the development of this container.

Looking forward to your feedback and hoping some of you would be interested in collaborating on this project.

Best regards,

Sam

**Comment by empiricompany** [comment]
We already have this Docker stack: https://github.com/OpenMage/magento-lts/tree/main/dev/openmage
Which specific settings were you thinking about on MySQL? 
They depend a lot on specific situations, but we could consider to link a configuration file template with the most common settings.
